src/styles/utils/variables.scss

```diff
@@ -0,0 +1,10 @@
+$cell-size: 100px;
```

Comment:
Consider using a more descriptive variable name instead of $cell-size, such as $calendar-cell-size.